samples/application-insights/sample.md

```diff
@@ -0,0 +1,82 @@
+
+This sample reports the following metrics to Application Insights:
+
+ * Fetched messages per second 
+ * Failed messages per second
+ * Successful messages per second
+ * Average message processing time in milliseconds
+ * Maximum critical time in milliseconds
+
+For a detailed explanation of these metrics refer to the [metrics captured section in the metrics documentation](/nservicebus/operations/metrics.md#metrics-captured).
+
+
+## Prerequisites
+
+An Application Insights account and instrumentation key. The instrumentation key can be obtained under the Application Insights instance properties pane in the Azure portal.
+
+
+## Code overview
+
+The sample simulates messages load with a random 10% failure rate using the `LoadSimulator` class:
+
+snippet: load-simulator
+
+
+## Connect to Azure Application Insights
+
+To connect the sample code to Application Insights, the instrumentation key needs to be provided.
+
+By default the sample code loads this key from an environment variable called `ApplicationInsightKey`. Either set this environment variable or paste the instrumentation key in the following section.
+
+snippet: configure-ai-instrumentation-key
+
+## Use NServiceBus.Metrics to capture metric values
+
+To capture the metric values generated by NServiceBus, a class called `ApplicationInsightsFeature` is added to the sample. It is responsible for enabling the NServiceBus metrics feature.
+
+snippet: enable-nsb-metrics
+
+It ensures that the telemetry probe is subscribed to the event stream. During this stage the `ApplicationInsightsProbeCollector` is configured and wired. Information such as *endpoint name* and *input queue* are passed to allow advanced data analysis in Application Insights.  
+
+snippet: register-probe
+
+Events and metrics are buffered and the Application Insights telemetry client does this every minute. If the endpoint instance would gracefully stop it needs to ensure that any pending events are flushed to Application Insight.
+
+snippet: flush-probe
+
+## Send captured metric data to Application Insights
+
+The `ApplicationInsightsProbeCollector` is used to pass NServiceBus captured metric data to the Application Insights instance configured earlier. Information passed is added as telemetry context properties, which will be attached to every metric (NServiceBus signal probe) and event (NServiceBus duration probe) captured via the telemetry client. The telemetry client is used to communicate with Application Insights.
+
+snippet: telemetry-client
+
+When the `ApplicationInsightsProbeCollector` is registered with the NServiceBus metrics API, it registers observer callbacks to receive probe names, signals and durations. Signals are converted into Application Insights custom events. Durations are converted into Application Insights metrics.
+
+snippet: observers-registration
+
+
+## Setting up charts for metrics and events (signals and durations)
```

Comment:
Very long heading. "Creating charts" would probably do.

Also notice you use the word "graphs" in the next paragraph - be consistent. Based on my google search the terms overlap A LOT but "chart" is probably the more accurate term for what we're dealing with.